include/turtle/detail/action.hpp

```diff
@@ -96,77 +84,54 @@ namespace detail
         template< typename Value >
         void returns( const Value& v )
         {
-            this->set( boost::ref( store( v ) ) );
+            this->set( std::ref( store( v ) ) );
         }
         template< typename Y >
-        void returns( const boost::reference_wrapper< Y >& r )
+        void returns( const std::reference_wrapper< Y >& r )
         {
             this->set( r );
         }
 
         template< typename Value >
-        void moves( BOOST_RV_REF(Value) v )
+        void moves( Value&& v )
         {
-            this->set(
-                boost::bind(
-                    &move< typename boost::remove_reference< Value >::type >,
-                    boost::ref( store( boost::move( v ) ) ) ) );
+            auto vRef = std::ref( store( std::move( v ) ) );
```

Comment:
`vRef` -> `vref`?

Reply:
Wanted to camel case this. It's a ref of v